apps/video/js/thumbnail_item.js

```diff
@@ -96,13 +96,13 @@ function ThumbnailItem(videoData) {
     }
 
     // popular html text
```

Comment:
While you're making changes to this file, if you could correct this comment -- "popular" --> "populate" -- that would be great.